src/store/reducer/user.js

```diff
@@ -118,10 +118,10 @@ function rootReducer(state = defaultState, action) {
     case "C9":
       const newArray4 = state.subjdata.filter((user) => {
         return (
-          user.class1 == "9" ||
-          user.class2 == "9" ||
-          user.class3 == "9" ||
-          user.class4 == "9"
+          user.class1 === "9" ||
+          user.class2 === "9" ||
+          user.class3 === "9" ||
+          user.class4 === "9"
```

Comment:
Could you do something with these repetitive lines of code? For each `case C9 C10 C11 C12`

Reply:
Don't worry one of the mentors would help you with this soon.

Reply:
Hey! @sagar-barapatre please help her.

Reply:
Yeah! This works absolutely fine. I have checked it... 
Great job @mitalirs !!

Reply:
@mitalirs facing any issue? Discord project channels are open now.

Reply:
Yes, I checked the channels. Actually, I wanted to add the shortened code to the pull request that I had created a few days back, but I am not sure about the best way of doing that. Once I figure that out I will update the PR.